src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -329,12 +329,13 @@
                  :pointer-events :box-none}
         [rn/view {:style {:width 40}}
          (when (or (and (seq response-to) (:quoted-message message)) last-in-group? pinned)
-           [user-avatar/user-avatar {:full-name         display-name
-                                     :profile-picture   photo-path
-                                     :status-indicator? true
-                                     :online?           true
-                                     :size              :small
-                                     :ring?             false}])]
+           [react/touchable-highlight {:on-press #(re-frame/dispatch [:chat.ui/show-profile from])}
```

Comment:
rf/ ? 🤔

Reply:
old one, rf/ will be replaces when moved to status-im2